pages/account.tsx

```diff
@@ -33,12 +38,12 @@ export default function PageAccount() {
 
         <div className="container">
           <div className="row justify-content-center">
-            <div className="col-md-10">
-              <IssueListItem></IssueListItem>
-            </div>
-            <div className="col-md-10">
-              <IssueListItem></IssueListItem>
-            </div>
+            {myIssues.map((issue) => (
```

Comment:
can we make a component out of this? from experience, writing this small loop will get old pretty quick. And we will start making it differently because we will get tired of writing it.